src/python/foglamp/core/server.py

```diff
@@ -59,12 +129,36 @@ def start(cls):
                 signal_name,
                 lambda: asyncio.ensure_future(cls._stop(loop)))
 
+        # TODO: make management port dynamic?!
+        host_address = "localhost"  # fix?!
+        core_mgt_port = 8082
+
+        # start scheduler
```

Comment:
Aren't we supposed to start the management webserver first? Otherwise, how do we start the scheduler?